src/main/java/seedu/address/storage/JsonAdaptedSession.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.session.Duration;
+import seedu.address.model.session.Fee;
+import seedu.address.model.session.Session;
+import seedu.address.model.session.SessionDate;
+import seedu.address.model.session.Subject;
+
+public class JsonAdaptedSession {
+    private final String sessionDate;
+    private final String duration;
+    private final String subject;
+    private final String fee;
+
+    /**
+     * Test
+     * @param sessionDate
+     * @param duration
+     * @param subject
+     * @param fee
+     */
+    @JsonCreator
+    public JsonAdaptedSession(@JsonProperty("sessionDate") String sessionDate, @JsonProperty("duration")String duration,
```

Comment:
whitespace missing after `@JsonProperty`

Reply:
I think that's how it's supposed to be, just like in `JsonAdaptedStudent`